### ISSUE TITLE
luma/chroma subsampling JPEG error

### ISSUE THREAD

**Issue by mike-hart** [imgproxy/imgproxy#207]
On 2.4.1 with the attached image:
```
jpeg.UnsupportedError: unsupported JPEG feature: luma/chroma subsampling ratio
  File "errors_reporting.go", line 70, in reportError
  File "server.go", line 111, in handlePanic
  File "router.go", line 80, in func1
  File "processing_handler.go", line 123, in handleProcessing
  File "server.go", line 93, in func1
  File "server.go", line 80, in func1
  File "router.go", line 91, in ServeHTTP
  File "server.go", line 2774, in ServeHTTP
  File "server.go", line 1878, in serve
```

![luma-chroma_subsampling_issue](https://user-images.githubusercontent.com/4129199/65252036-32280300-daf0-11e9-95c0-b0fa3bf29dbf.jpg)

**Comment by mike-hart** [comment]
`latest` (as of now) works, but nowhere near as bright/vibrant as my browser renders the above directly.

**Comment by DarthSim** [comment]
This is a CMYK image, and its rendering may vary in different viewers. For example, macOS Preview and LilyView render it the same way as imgproxy, but Photoshop renders it just like Chrome.

**Comment by DarthSim** [comment]
Fixed in [2.6.0](https://github.com/imgproxy/imgproxy/releases/tag/v2.6.0)
